db-recipe-book/data/migrations/20181011173917_recipes.js

```diff
@@ -0,0 +1,16 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('recipes', function(tbl){
+        tbl.increments();
+        tbl.string('name', 50).notNullable();
+        tbl.integer('recipe_id').unsigned();
+        tbl.foreign('recipe_id').references('dishes.dish_id');
```

Comment:
Again, you don't need a `recipe_id` since there is already an `id` column for each recipe. You should also name this column `dishes_id` for the first part of line 7, and then reference `dishes.id` for the second part of line 7.